app/routes/auth.py

```
@@ -0,0 +1,271 @@
+        users = await get_all_users(session)
+        return {
+            "users": users
+        }
+    
+    except Exception as e:
+        logger.exception("Unhandled exception in get users endpoint")
+        raise InternalServerError("An unexpected error occurred while retrieving users.") from e
+@router.delete("/users/{uid}")
+@limiter.limit("5/minute")  # Custom limit for this endpoint
+async def delete_user_route(
+    request: Request,
```

Ruff (0.12.2)

Unused function argument: `request`

(ARG001)